src/lib/services/identity.ts

```diff
@@ -1,8 +1,43 @@
 import { NotFound, HttpResponse } from '../error-handler'
 import env from '../../env'
+import { Status, serviceState } from '../service-watcher/statusPoll'
 
 const URL_PREFIX = `http://${env.IDENTITY_SERVICE_HOST}:${env.IDENTITY_SERVICE_PORT}/v1`
 
+export default class IdentityClass {
+  constructor() {}
+
+  getStatus = async (): Promise<Status> => {
+    try {
+      const res = await getHealth()
+      if (res) {
+        if (res.version.length < 1) {
```

Comment:
This is ok but what might be better is a semver regex check that it is a valid version?